staging/src/k8s.io/kubectl/pkg/cmd/logs/logs.go

```diff
@@ -134,10 +136,9 @@ type LogsOptions struct {
 	containerNameFromRefSpecRegexp *regexp.Regexp
 }
 
-func NewLogsOptions(streams genericiooptions.IOStreams, allContainers bool) *LogsOptions {
+func NewLogsOptions(streams genericiooptions.IOStreams) *LogsOptions {
 	return &LogsOptions{
```

Comment:
could you tell me why remove "allContainers"?

Reply:
There's some context on the previous iteration of this PR.

https://github.com/kubernetes/kubernetes/pull/123874#discussion_r1548680888

TLDR: It shouldn't have been added to the signature in the first place.